plugins/catalog-react/src/components/EntityOwnerPicker/useFacetsEntities.ts

```diff
@@ -52,37 +49,30 @@ export function useFacetsEntities({ enabled }: { enabled: boolean }) {
       return [];
     }
     const facet = 'relations.ownedBy';
-    const facetsResponse = await catalogApi.getEntityFacets({
-      facets: [facet],
-    });
-    const entityRefs = facetsResponse.facets[facet]?.map(e => e.value) ?? [];
 
     return catalogApi
-      .getEntitiesByRefs({ entityRefs })
-      .then(resp =>
-        resp.items
-          .filter(entity => entity !== undefined)
-          .map(entity => entity as Entity)
+      .getEntityFacets({ facets: [facet] })
+      .then(response =>
+        response.facets[facet]
+          .map(e => e.value)
+          .map(ref => {
+            const { kind, name, namespace } = parseEntityRef(ref);
```

Comment:
so this will actually create a broken entity (potentially wrong casing for fields, definitely the kind)